### PR TITLE
Fix a command execution timeout failure

### PR DESCRIPTION
**Pull request by nanzhg**
Fix a setup command failure for kubevirt test - http://10.67.129.96/tests/3322#step/kubevirt_tests_server/276

- Related ticket: https://progress.opensuse.org/issues/160910
- Verification run: http://10.67.129.96/tests/3344

### CODE REVIEW

File: tests/virt_autotest/kubevirt_tests_server.pm [hunk 1]
```diff
@@ -489,7 +489,7 @@ EOF
     # Workaround for bsc#1199448
     my $node_helper = 'node-helper.yaml';
     assert_script_run("curl " . data_url("virt_autotest/kubevirt_tests/$node_helper") . " -o $node_helper");
-    assert_script_run("sed -e \"s/$kubevirt_ver/\${KUBEVIRT_VERSION}/g\"");
+    assert_script_run("sed -i 's/\${KUBEVIRT_VERSION}/$kubevirt_ver/g'");
```
Comment (alice-suse) [inline tests/virt_autotest/kubevirt_tests_server.pm:492]: No file name given to sed?

Comment (Julie-CAO) [inline tests/virt_autotest/kubevirt_tests_server.pm:492]: No filename specified in `sed` command, does it work?

Comment (nanzhg) [reply tests/virt_autotest/kubevirt_tests_server.pm]: Sorry, my mistake. Updated.

Comment (nanzhg) [reply tests/virt_autotest/kubevirt_tests_server.pm]: Sorry, my mistake. Updated.

**Comment by alice-suse** [comment]
Please let me know when new quick verification job is done :)

**Comment by nanzhg** [comment]
> Please let me know when new quick verification job is done :)

The failure got resolved in the latest verification run, I've updated the job link. Thanks :)

**Pull request merged**